proto/accessgraph/v1/query.proto

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package accessgraph.v1;
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/accessgraph/v1;accessgraphv1";
+
+// QueryRequest is a request to query the access graph.
+message QueryRequest {
+  string query = 1;
+}
+
+// Node is a node in the access graph.
```

Comment:
What can a node represent?

Reply:
Node can represent anything. I'm not sure how much domain specific knowledge should proto carry compare to docs TBH.

Reply:
At the bare minimum enough for people looking at the protos do have an idea of what it can do, but if you ask me this is the place for dev-focused docs.